packages/web-react/src/components/Toast/demo/ToastStaticToast.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button } from '../../Button';
+import Toast from '../Toast';
+import ToastBar from '../ToastBar';
+
+const ToastStaticToast = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
```suggestion
  import { useState } from 'react';
  const [isOpen, setIsOpen] = useState(false);
```

Just minor adjustment.

Reply:
fixed